src/amo/components/Search/SearchResult.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import './SearchResult.scss';
+
+export default class SearchResult extends React.Component {
+  static propTypes = {
+    result: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { result } = this.props;
+    return (
+      <li className="SearchResult">
+        <Link to={`/en-GB/firefox/addon/${result.slug}/`}
+              className="SearchResult-link"
+              ref={(el) => { this.name = el; }}>
+          <div>
+            <img className="SearchResult-icon" src={result.icon_url}
+                 alt="Icon" />
+          </div>
+          <section className="SearchResult-main">
+            <h2 className="SearchResult-heading">
+              <span className="SearchResult-name">
```

Comment:
Are we needing this span for CSS?

Presumably the search result is just the heading ahead of showing more information?